internal/cli/cmd/connectivity.go

```diff
@@ -172,9 +172,10 @@ func newCmdConnectivityTest(hooks Hooks) *cobra.Command {
 
 	initSysdumpFlags(cmd, &params.SysdumpOptions, "sysdump-", hooks)
 
-	cmd.Flags().BoolVar(&params.IncludeInterruptionTest, "include-interruption-test", false, "Include interruption test (upgrade test, ipsec key rotation test, etc.)")
+	cmd.Flags().BoolVar(&params.InterruptionTestForUpgrade, "interruption-test-for-upgrade", false, "Interruption test for upgrade")
+	cmd.Flags().BoolVar(&params.InterruptionTestForIPsecKeyRotation, "interruption-test-for-ipsec-key-rotation", false, "Interruption test for IPsec key rotation")
```

Comment:
The reason why I renamed the `--include-int-test` to `--int-test-for-upgrade` was I could make it more concise:

- Option A: `--include-int-test` for normal upgrade test, `--include-int-test --include-ipsec-int-test` for IPsec upgrade test
- Option B: `--int-test-for-upgrade` for normal one, `--int-test-for-ipsec` for ipsec

I chose option A in the beginning, then I thought option B is better. Option A also leaves the solo flag `--include-ipsec-int-test` (without `--include-int-test`) unclear. 

If the concern is rename, how about Option C:

- Option C: `--include-upgrade-test` for normal, `--include-ipsec-upgrade-test` for ipsec

This way we can even keep `--include-upgrade-test` untouched.

Reply:
I'm asking why we even need multiple flags.

I'm proposing to have only `--include-int-test` and put the IPsec-specific logic behind a `if IsEnabled(FeatureIPsec)` condition. I'm proposing this because I don't see anything in the test that is specific to upgrades or to key rotations (maybe I missed it). It's just a bunch of checks, with some (XFRM counters) that should only be performed when IPsec is enabled.

Reply:
True! Let me make it right.